src/argo.h

```diff
@@ -47,5 +47,25 @@ int  argo_node_id();
  * @return The total number of ArgoDSM nodes
  */
 int  argo_number_of_nodes();
+/**
```

Comment:
consistency with the rest of the file would suggest to have an empty line here (applies multiple times in this file)